benchmarks/profiler/utils/search_space_autogen.py

```
@@ -0,0 +1,87 @@
+    # first check if config file exists
+    if args.model is not None:
+        if not args.config:
+            # modify config file from default config file
+            logger.info("DGD config file not provided, using default config file")
+            config = config_modifier.load_default_config()
+
+            logger.info(
+                f"Saving generated DGDconfig file to {args.output_dir}/disagg_config.yaml"
+            )
+        else:
+            config = yaml.load(args.config)
```

Ruff (0.14.1)

Probable use of unsafe `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)